packages/coinstac-client-core/src/index.js

```diff
@@ -237,6 +237,19 @@ class CoinstacClient {
       });
   }
 
+  /**
+   * Stop pipeline
+   * @param {string} pipelineId The id of the pipeline running
+   * @param {string} runId The id of the pipeline run
+   */
+
+  stopPipeline( pipelineId, runId ) {
+    const stopPipeline = this.pipelineManager.stopPipeline(pipelineId, runId);
+    return new Promise((res, rej) => {
```

Comment:
Why create a new promise here? Also, this new promise doesnt resolve or reject anything, so you'll never actually get the async data to return right. `stopPipeline` should return a promise and thus you can just make this a one liner es6 arrow function